Source/HL2Editor/Private/BSPImporter.cpp

```diff
@@ -150,25 +150,25 @@ bool FBSPImporter::ImportEntitiesToWorld(UWorld* targetWorld)
 		FHL2EntityData entityData;
 		entityData.Classname = fnStaticProp;
 		entityData.Origin = FVector3f(staticProp.m_Origin(0, 0), staticProp.m_Origin(0, 1), staticProp.m_Origin(0, 2));
-		entityData.KeyValues[fnSolidity] = FString::FromInt((int)staticProp.m_Solid);
-		const char* modelStr = bspFile.m_StaticpropStringTable[staticProp.m_PropType].m_Str;
-		const auto& modelRaw = StringCast<TCHAR, 128>(modelStr);
-		entityData.KeyValues[fnModel] = FString(modelRaw.Length(), modelRaw.Get());
-		entityData.KeyValues[fnAngles] = FString::Printf(TEXT("%f %f %f"), staticProp.m_Angles(0, 0), staticProp.m_Angles(0, 1), staticProp.m_Angles(0, 2));
-		entityData.KeyValues[fnSkin] = FString::FromInt(staticProp.m_Skin);
+		entityData.KeyValues.Add(fnSolidity, FString::FromInt((int)staticProp.m_Solid));
+		const char *modelStr = bspFile.m_StaticpropStringTable[staticProp.m_PropType].m_Str;
+		const auto &modelRaw = StringCast<TCHAR, 128>(modelStr);
```

Comment:
Can you keep the existing pointer/reference style in line with the rest of the project?

```suggestion
		const char* modelStr = bspFile.m_StaticpropStringTable[staticProp.m_PropType].m_Str;
		const auto& modelRaw = StringCast<TCHAR, 128>(modelStr);
```

Reply:
My bad it must have been auto format.

Reply:
```suggestion
		const char *modelStr = bspFile.m_StaticpropStringTable[staticProp.m_PropType].m_Str;
		const auto &modelRaw = StringCast<TCHAR, 128>(modelStr);
```